packages/Webkul/CartRule/src/Http/Controllers/CartRuleController.php

```diff
@@ -95,9 +95,9 @@ public function copy(int $cartRuleId): View
 
     /**
      * Store a newly created resource in storage.
-     * 
+     *
      * @param  \Webkul\CartRule\Http\Requests\CartRuleRequest  $cartRuleRequest
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Http\RedirectResponse
```

Comment:
In the end, `RedirectResponse` is already extended from `Response`. So revert this one.

Reply:
No extend from `Symfony RedirectResponse` !!